packages/website/src/components/inputs/Tooltip.tsx

```diff
@@ -4,18 +4,20 @@ import React from 'react';
 import styles from './Tooltip.module.css';
 
 export interface TooltipProps {
-  readonly children: JSX.Element | (JSX.Element | false)[];
+  readonly children: React.ReactNode;
   readonly text: string;
   readonly position?: 'left' | 'right';
   readonly open?: boolean;
   readonly hover?: boolean;
+  readonly clasName?: string;
```

Comment:
[Typo]

```suggestion
  readonly className?: string;
```

Aside: I wonder how many KB the internet would be smaller by if this was how it was spelled...